clients/scanoss/src/main/kotlin/model/Model.kt

```diff
@@ -105,6 +105,52 @@ enum class Source {
     SCANCODE
 }
 
+/**
+ * The "vulnerabilities" section of the raw report.
+ */
+@Serializable
+data class Vulnerability(
```

Comment:
Also here, the commit message should say

    scanoss: Introduce a dedicated `Vulnerability` class